pkg/apis/hco/v1alpha1/hyperconverged_types.go

```diff
@@ -16,6 +16,11 @@ type HyperConvergedSpec struct {
 	KubeVirtImagePullPolicy      string `json:"KubeVirtImagePullPolicy,omitempty"`
 	CDIImagePullPolicy           string `json:"CDIImagePullPolicy,omitempty"`
 	NetworkAddonsImagePullPolicy string `json:"NetworkAddonsImagePullPolicy,omitempty"`
+	KWebUIImagePullPolicy        string `json:"KWebUIImagePullPolicy,omitempty"`
```

Comment:
Don't add any more config options.  We decided that the HCO is going to have none in the first release.  Your operator should default all these values.

Reply:
Does it mean that for different branding (like for downstream) we will need to patch this golang code and produce separate build?

Reply:
In addition, the `KWebUIPublicMasterHostname` and `KWebUIMasterDefaultSubdomain` are expected to be provided if `openshift-console` is not deployed. Otherwise the web-ui will be inaccessible. Any suggestion how to provide these params for that flow?

Reply:
So this would be the case where openshift is deployed without the webui?

Reply:
That would be the case when openshift is deployed without console (the default, original Web UI - the github.com/openshift/console project). Which is a non-default but still valid case.

Subsequently, Kubevirt will deploy the kubevirt-web-ui which needs to be provided for these params, as they can not be auto-discovered by reading openshift-console configuration.

Reply:
No matter of these two params, the issue with branding is still there.

Reply:
do these params need to be changed at run time or will setting them downstream be enough?

Reply:
Values for these params are specific for particular setup - it's part of URL the `route` is composed from.
Again: they are required for the flow _without_ openshift-console installed.